frontend/packages/ceph-storage-plugin/src/components/modals/clone-pvc-modal/clone-pvc-modal.tsx

```diff
@@ -0,0 +1,167 @@
+
+  const pvcUsedCapacityQuery: string = getPVCUsedCapacityQuery(pvcName);
+
+  const [response, error, loading] = usePrometheusPoll({
+    endpoint: PrometheusEndpoint.QUERY,
+    query: pvcUsedCapacityQuery,
+    namespace: resource.metadata.namespace,
+  });
+
+  const pvcUsedCapacityQueryResult: DataPoint[] = getInstantVectorStats(
+    response,
+    null,
+    humanizeBinaryBytes,
+  );
+
+  const pvcUsedCapacity = pvcUsedCapacityQueryResult?.[0]?.label || 'No Data';
+
+  const submit = (event: React.FormEvent<EventTarget>) => {
+    event.preventDefault();
+
+    const pvcCloneObj = {
+      apiVersion: PersistentVolumeClaimModel.apiVersion,
+      kind: PersistentVolumeClaimModel.kind,
+      metadata: {
+        name: clonePVCName,
+        namespace: resource.metadata.namespace,
+      },
+      spec: {
+        storageClassName: resource.spec.storageClassName,
+        dataSource: {
+          name: pvcName,
+          kind: PersistentVolumeClaimModel.kind,
+          apiGroup: '',
+        },
+        accessModes: resource.spec.accessModes,
+        resources: resource.spec.resources,
+      },
+    };
+
+    return handlePromise(k8sCreate(PersistentVolumeClaimModel, pvcCloneObj)).then(close);
+  };
+
+  return (
+    <Form onSubmit={submit}>
+      <div className="modal-content modal-content--no-inner-scroll">
+        <ModalTitle>Clone</ModalTitle>
+        <ModalBody>
+          <FormGroup label="Name" isRequired fieldId="ceph-clone-pvc-modal__name">
+            <TextInput
+              isRequired
+              type="text"
+              name="ceph-clone-pvc-modal__name"
+              value={clonePVCName}
+              onChange={setClonePVCName}
+            />
+          </FormGroup>
+          <div className="ceph-clone-pvc-modal__details">
+            <label htmlFor="ceph-clone-pvc-modal__details-section" className="text-muted">
+              PVC Details
+            </label>
+            <div className="ceph-clone-pvc-modal__details-section">
+              <div>
+                <div>
+                  <label htmlFor="ceph-clone-pvc-modal__namespace">Namespace</label>
+                  <p>
+                    <ResourceIcon kind="Namespace" />
+                    {resource.metadata.namespace}
+                  </p>
+                </div>
+                <div>
+                  <label htmlFor="ceph-clone-pvc-modal__storageclass">Storage Class</label>
+                  <p>
+                    {resource.spec.storageClassName ? (
+                      <>
+                        <ResourceIcon kind="StorageClass" />
+                        {resource.spec.storageClassName}
+                      </>
+                    ) : (
+                      'None'
+                    )}
+                  </p>
+                </div>
+              </div>
+              <div>
+                <div>
+                  <label htmlFor="ceph-clone-pvc-modal__requested-capacity">
```

Comment:
No element with this ID. You should remove the `htmlFor`

Reply:
removing `htmlFor` is giving linting error.